wrappers/services.go

```diff
@@ -456,3 +458,88 @@ func renderListenStream(socket *snap.SocketInfo) string {
 	listenStream := strings.Replace(socket.ListenStream, "$SNAP_DATA", snap.DataDir(), -1)
 	return strings.Replace(listenStream, "$SNAP_COMMON", snap.CommonDataDir(), -1)
 }
+
+func makeAbbrevWeekdays(start time.Weekday, end time.Weekday) []string {
```

Comment:
Ahm, why not just 
```
var weekDaysAbbr := []string{"Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"}
```

and a simple slice below?

Reply:
Simple slice will not work unless it's doubled, eg. `{Sun, Mon.., Sat, Sun, Mon... ,Sat}` You can have a span such as Fri-Mon, what's supposed to yield `{Fri, Sat, Sun, Mon}`. I can make a slice/map of abbreviated weekday names, so this will avoid `Weekday(..).String()[0:3]`.

Also note that this piece runs only when generating *.timer file, meaning once in a while, when installing a snap with timer services.